doc/source/cluster/kubernetes/user-guides/gke-gcs-bucket.md

```diff
@@ -0,0 +1,168 @@
+# Configuring KubeRay to use Google Cloud Storage Buckets in GKE
```

Comment:
Add `(XXXXXXX)=` in this doc and refer it in `user-guides.md`.

Reply:
Thanks, done!